run_ssps/Snakefile

```diff
@@ -81,5 +81,6 @@ rule run_mcmc:
     shell:
         "julia --project={JULIA_PROJ_DIR} {input.method} {input.ts_file} {input.ref_dg} {output} {TIMEOUT}"\
         +" --regression-deg {REGDEG} --n-steps {MAX_SAMPLES}"\
-        +" --lambda-prop-std {LSTD} --large-indeg {LARGE_INDEG}"
+        +" --lambda-prop-std {LSTD} --large-indeg {LARGE_INDEG}"\
+        +" --multithread"
```

Comment:
Do we need to document in the readme how the `snakemake --cores` argument interacts with SSPS multithreading?

Reply:
Yes, I will add that to the README. 

FYI, the parameters multiply: i.e, if we run `snakemake --cores 2` and have the environment variable `JULIA_NUM_THREADS=3`, then the SSPS jobs may use <= 6 CPUs at a time. I don't think this is a good design. But I was constrained by the way parallelism works in Julia.

It looks like the next Julia version (v1.5) allows the number of threads to be set via command line argument, rather than environment variable. Which will allow Snakemake and Julia to coordinate parallelism better.